hardware/USBtin_MultiblocV8.h

```diff
@@ -0,0 +1,216 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+#include "DomoticzHardware.h"
+#include "hardwaretypes.h"
+
+#define	TIME_1sec				1000000

```

Comment:
Is it possible to move the defines to the top of the cpp file ?

Reply:
ok !